curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -361,19 +363,27 @@ int FileCacheManager::Read(uint64_t inodeId, uint64_t offset, uint64_t length,
             if (ret < 0) {
                 retry++;
                 responses.clear();
-                if (ret != -2 || retry == maxRetry) {
+                if (ret != -2) {
```

Comment:
what does ret==-2 mean?

Reply:
This is the linux error code, which means ENOENT，indicating that the data cannot be read on s3